inc/critical-style.php

```diff
@@ -18,6 +18,6 @@ function nightingale_critical_styles() {
 
 
 $theme_colour = get_theme_mod( 'theme_colour', 'nhs_blue' );
-if ( 'nhs_blue' === $theme_colour || '' === $theme_colour ) { // Default NHS Blue - load up the critical CSS inline, Boom.
+if ( '005eb8' === $theme_colour || 'nhs_blue' === $theme_colour || '' === $theme_colour ) { // Default NHS Blue - load up the critical CSS inline, Boom.
```

Comment:
That hardcoded '005eb8' looks odd, I guess in case of a colour change it won't work, so perhaps it's worth of a variable assignment in sass